README.md

```diff
@@ -1,76 +1,5 @@
-![](https://img.shields.io/badge/Microverse-blueviolet)
+# FCC Test Suite Template
 
-# Project Name
+**Fork** this project (do not open a Pull Request) to start working on any of freeCodeCamp's responsive web design course projects. Select the project from the dropdown menu and run the tests to see which requirements your project is still missing.
 
-> One paragraph statement about the project.
-
-![screenshot](./app_screenshot.png)
-
-Additional description about the project and its features.
-
-## Built With
-
-- Major languages
-- Frameworks
-- Technologies used
-
-## Live Demo
```

Comment:
Please, make this live demo functional.

Reply:
* KIndly state why this change was requested as this helps the student understand why and learn from it 👍

Reply:
- Please make this live demo functional in order for viewers to can check your work until now